src/Symfony/Component/Security/Core/Authentication/RememberMe/PersistentToken.php

```diff
@@ -18,18 +18,55 @@
  */
 final class PersistentToken implements PersistentTokenInterface
 {
+    private ?string $class = null;
+    private string $userIdentifier;
+    private string $series;
+    private string $tokenValue;
     private \DateTimeImmutable $lastUsed;
 
     public function __construct(
-        private string $class,
-        private string $userIdentifier,
-        private string $series,
-        #[\SensitiveParameter] private string $tokenValue,
```

Comment:
we should keep the attribute on both the new and the old positions for now

Reply:
oops, indeed 👍